frontend/app/routes/_public+/apply+/$id+/terms-and-conditions.tsx

```diff
@@ -0,0 +1,173 @@
+import { ActionFunctionArgs, LoaderFunctionArgs, json, redirect } from '@remix-run/node';
+import { useLoaderData } from '@remix-run/react';
+
+import { useTranslation } from 'react-i18next';
+
+import { ButtonLink } from '~/components/buttons';
+import { CollapsibleDetails } from '~/components/collapsible';
+import { InlineLink } from '~/components/inline-link';
+import { getApplyFlow } from '~/routes-flow/apply-flow';
+import { getTypedI18nNamespaces } from '~/utils/locale-utils';
+import { RouteHandleData } from '~/utils/route-utils';
+
+const i18nNamespaces = getTypedI18nNamespaces('terms-and-conditions');
+
+export const handle = {
+  breadcrumbs: [
+    {
+      labelI18nKey: 'terms-and-conditions:terms-and-conditions.index.breadcrumbs.terms-and-conditions',
+    },
+  ],
+  i18nNamespaces,
+  pageTitleI18nKey: 'terms-and-conditions:terms-and-conditions.index.page-title',
+} as const satisfies RouteHandleData;
+
+export async function loader({ request, params }: LoaderFunctionArgs) {
+  const applyFlow = getApplyFlow();
+  const { id, state } = await applyFlow.loadState({ request, params });
+  return json({ id, state: state.termsAndConditions });
+}
+
+export async function action({ request, params }: ActionFunctionArgs) {
+  const applyFlow = getApplyFlow();
+  const { id } = await applyFlow.loadState({ request, params });
+
+  const sessionResponseInit = await applyFlow.saveState({
+    request,
+    params,
+    state: {},
+  });
+
+  return redirect(`/apply/${id}/personal-info`, sessionResponseInit);
+}
+
+export default function TermsAndConditions() {
+  const { id } = useLoaderData<typeof loader>();
+  const { t } = useTranslation(i18nNamespaces);
+  return (
+    <>
+      <p className="text-xl">{t('terms-and-conditions:terms-and-conditions.index.intro-text')}</p>
+      <br />
+      <CollapsibleDetails id="collapsable-description-first-part" summary={t('terms-and-conditions:terms-and-conditions.index.terms-and-conditions-of-use-title')}>
+        <p>{t('terms-and-conditions:terms-and-conditions.index.terms-and-conditions-of-use-part-one')}</p>
+        <br />
```

Comment:
Avoid using `<br />` for spacing. Use [margins](https://tailwindcss.com/docs/margin) on the `<p>` instead (eg. `<p className="mb-6">`).

This comment can be applied throughout this page where `<br />` is used.